src/main/java/Storage.java

```diff
@@ -0,0 +1,139 @@
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+
+/**
+ * Makeshift database that stores all task list in a file.
+ */
+public class Storage {
+
+    public static String filePath;
+
+    /**
+     * Creates a makeshift database in file at specified filepath.
+     * @param filePath path to file that stores the task list.
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Loads list of tasks from the database.
+     * @return list of tasks.
+     * @throws FileNotFoundException if file annot be found.
+     */
+    public List<Task> load() throws FileNotFoundException {
+        File f = new File(this.filePath); // create a File for the given file path
+        Scanner s = new Scanner(f); // create a Scanner using the File as the source
+        List<Task> list = new ArrayList<>();
+
+        while (s.hasNext()) {
+            String[] parts = s.nextLine().split(" \\| ");
+            switch (parts[0]) {
+            case "T":
+                list.add(new Todo(parts[2], Integer.parseInt(parts[1])));
+                break;
+            case "D":
+                list.add(new Deadline(parts[2], Integer.parseInt(parts[1]), convertDateFormat(parts[3])));
+                break;
+            case "E":
+                list.add(new Event(parts[2], Integer.parseInt(parts[1]), convertDateFormat(parts[3])));
+                break;
+            default:
+            }
+        }
+
+        return list;
+    }
+
+    /**
+     * Converts string indicating month to its corresponding integer as a string.
+     * @param month string of month.
+     * @return integer of month formatted as a string.
+     */
+    public String convertMonthToInt(String month) {
```

Comment:
Okay not very sure but for me I just accepted one date format, then you can just parse the String with LocalDate class and it should work, so you don't have to do these switch case things

Reply:
Hello! Do you know what is the LocalDate method to parse the month string?

Reply:
Yup! I think you can use the link here: https://www.geeksforgeeks.org/localdate-parse-method-in-java-with-examples/, then just put "MMM" as your pattern? I think it should work